internal/stack/_static/kibana.yml.tmpl

```diff
@@ -125,6 +125,20 @@ xpack.fleet.outputs:
     ca_trusted_fingerprint: "${ELASTIC_PACKAGE_CA_TRUSTED_FINGERPRINT}"
     is_default: true
     is_default_monitoring: true
+    {{- if semverLessThan $version "8.12.0" }}
```

Comment:
Added support for preset field in Kibana [here](https://github.com/elastic/kibana/pull/172359) (Kibana >=8.12)